src/lambdas/inspect/index.js

```diff
@@ -11,8 +11,9 @@ import { inspect as video } from "./video.js";
 
 /**
  * @typedef {object} InspectTask
- * @property {string} Type
+ * @property {string} [Type]
  * @property {boolean} [EBUR128]
+ * @property {string} [MatchTags]
```

Comment:
Added a Task attribute to pass in a regex that can be used to select and return only certain tags